pkg/testgridanalysis/testgridconversion/ocp_synthentic_tests.go

```diff
@@ -138,7 +138,7 @@ func (openshiftSyntheticManager) CreateSyntheticTests(rawJobResults testgridanal
 					syntheticTests[testgridanalysisapi.UpgradeTestName].pass = 1
 					// if the test succeeded, then the operator install tests should all be passes
 					for _, operatorState := range jrr.FinalOperatorStates {
-						testName := testgridanalysisapi.OperatorUpgradePrefix + " " + operatorState.Name
+						testName := testgridanalysisapi.OperatorUpgradePrefix + operatorState.Name
```

Comment:
the const already has a trailing space, so this wasn't needed.  I debated changing the const, but this seemed like it would have less unintended consequences.